### ISSUE TITLE
Error text contrast issue

### ISSUE THREAD

**Issue by houbly** [wmcadigital/wmn-design-system#286]
Error text doesn't pass contrast requirements on design system background

![image](https://user-images.githubusercontent.com/6360479/90389405-6f3ea180-e081-11ea-9b50-ea13f17580fa.png)

**Comment by mrmjprice** [comment]
@houbly - The colour that is being shown to you is not the same as what is being shown to me.

The colour you are shown in the screenshot is #D02F1B, the colour used in the CSS is WAG 2.1 AA compliant: #D4351C

**Comment by houbly** [comment]
> @houbly - The colour that is being shown to you is not the same as what is being shown to me.
> 
> The colour you are shown in the screenshot is #D02F1B, the colour used in the CSS is WCAG 2.1 AA compliant: #D4351C

@mrmjprice Yeah its just that the design system uses #F3F2F1 as the background colour which fails. It's not a big deal but Dayle said I should probably log it anyway.

**Comment by daylesalmon** [comment]
Relates to #287

**Comment by daylesalmon** [comment]
Relates to: https://github.com/wmcadigital/wmn-sprints/issues/36

**Comment by daylesalmon** [comment]
Closing this as a none-issue.
As we are using the correct hex colour and it seems as though it is WCAG 2.1 AA compliant.
